scss/components/_state-screen.scss

```diff
@@ -80,6 +81,13 @@
     top: 5%;
     left: 5%;
 
+    transition: 0.2s opacity ease-in-out, 0.3s transform ease-in-out;
```

Comment:
Sorry I missed this the last time. Is it deliberate to use a different duration for the transform transition?

Reply:
Ah I copied this from the skip controls css to keep things consistent:

https://github.com/ooyala/html5-skin/blob/master/scss/components/_skip-controls.scss#L8

Should I update both with 0.3s for the opacity transform? It seems like 0.3s is the go to transition/transform time elsewhere in our skin

Reply:
Oh ok sorry, I forgot I had done that, but it was deliberate. If possible it might be good to move this to a variable since they now both use the same transition, but  not super important

Reply:
I'll add this in